libs/accounts/recovery-phone/src/lib/recovery-phone.service.spec.ts

```diff
@@ -122,6 +122,34 @@ describe('RecoveryPhoneService', () => {
     expect(mockRecoveryPhoneManager.getAllUnconfirmed).toBeCalledWith(uid);
   });
 
+  it('handles localized message body when provided to setup phone number', async () => {
```

Comment:
nit: The fact it's 'localized' doesn't even matter in my opinion. I think a more agnostic way to think about this is that the message is 'formatted'. A localized string, or any other number of 'formatted' strings could be sent in here.